requirements.txt

```diff
@@ -1,2 +1,5 @@
 docutils>=0.12
 spdx-lookup>=0.3.0
+packaging
+setuptools
+FreeBSD-ports>=0.0.9
```

Comment:
Yep per @tykling these should go in 

`setup.py: install_requires = [`

Reply:
We should drop requirements.txt too, replacing the following line in README.md:

`$ pip install -r requirements.txt`

with

`$pip install pytoport`